python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -64,11 +64,11 @@
 def import_onnx_modules():
     """ To make sure ONNX is runtime dependency, it is imported used only when needed"""
     try:
-        from onnx import helper, numpy_helper, mapping
+        from onnx import helper, numpy_helper, mapping, TensorProto
```

Comment:
Why do the import this way? Why not do this at the beginning along with other import statements. This sort of import creates a lot of other unnecessary changes in other methods.

Reply:
here functions are invoked according to the operators in the model.. so try catch with import statement doesn't work if you just do it at the top. You will have to import with try catch in every method, so this way seems much better

Reply:
TensorProto is used only in one method. Would it make sense to move the import of Tensorproto to the respective method?

Reply:
Importing "onnx" in import_onnx_modules now instead of helper functions.